v2/cli/init_templates.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"bytes"
+	"text/template"
+)
+
+// Template project.yaml file for a non-git project
+// nolint: lll
+var newProjectFileBytes = []byte(`# yaml-language-server: $schema=https://raw.githubusercontent.com/brigadecore/brigade/v2/v2/apiserver/schemas/project.json
+apiVersion: brigade.sh/v2-alpha.5
+kind: Project
+metadata:
+  id: {{ .ID }}
+description: Your newly-initialized Brigade project!
+spec:
+  workerTemplate:
+    logLevel: DEBUG
```

Comment:
Let's comment this line. I don't want debug level logging on by default, but I do want people to very clearly see how it's done. So if it's there and commented, they can just uncomment it if they choose.